examples/v1beta1/metrics-collector/tfevent-metrics-collector.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: kubeflow.org/v1beta1
```

Comment:
The training code maintained in the training-operator repo use TF 1.X so I added this new example using TF 2.X apart from TFJob example.

If we do not keep to support TF 1.X in tfevent-metrics-collector similar to `enas` suggestion, I think we can use new tf-example as an example for TFJob instead of example using TF 1.X in the training-operator repo.

WDYT? @andreyvelich

Reply:
Sounds good, let's just modify TFJob example with our new image for TF Event metrics collector.

Reply:
I addressed in https://github.com/kubeflow/katib/pull/1731/commits/58fbac654988038a18dbbb00ab27e0dae4324430